.github/styles/Vocab/Products/accept.txt

```diff
@@ -270,6 +275,8 @@ toolchain
 Toolchains
 trie
 TXIDs
+Tx
+TX
```

Comment:
Not sure about these two. Should we make TX the only option?

Reply:
Resolved